about.html

```diff
@@ -0,0 +1,128 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Creeds</title>
+</head>
+
+<body>
+    <div class="top-links-container">
+        <div class="top-links flex">
+            <a href="#facebook"><i class="fa fa-facebook-f"></i></a>
+            <a href="#twitter"><i class="fa fa-twitter"></i></a>
+            <a href="">
+                <h4>English</h4>
+            </a>
+            <a href="">
+                <h4>My Page</h4>
+            </a>
+            <a href="">
+                <h4>Logout</h4>
+            </a>
+        </div>
+    </div>
+    <header>
+        <div class="nav-bar">
+            <div class="topnav">
+                <a class="logo" href="./index.html"><img src="./assests/header-logo.png" alt="logo" /></a>
+                <h3 class="brand-name">Creeds</h3>
+                <div class='topnav-right'>
+                    <a id="menu-button" class="menu-button" href="#"><i class="fa fa-bars"></i></a>
+                    <div class="hidden">
+                        <div class="mobile-menu flex">
+                            <i id="close-menu" class="fa fa-close"></i>
+                            <ul class="nav-items flex">
+                                <li class="list-style nav-list flex"><a class="nav-elem-active" href="./index.html">Home</a></li>
+                                <li class="list-style nav-list flex"><a class="nav-elem" href="#disapled">Program</a></li>
+                                <li class="list-style nav-list flex"> <a class="nav-elem" href="#disapled">News</a></li>
+                                <li class="list-style nav-list flex"> <a class="nav-elem" href="#disapled">Contact</a></li>
+                                <li class="list-style nav-list flex campaign">CC Campaign</li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main id="home">
+        <div class="home-container">
+            <div class="home">
+                <h3>"Hello Sharing World!!"</h3>
+                <div class="main-header">
+                    <h1 class="title-header">CREEDS<br/> COMPANY FOR CARS SPARES<br/>FOUNDED AT 2022</h1>
+                </div>
+                <div class="about-text-container">
+                    <p class="main-text">
+                        A startup created in 2022 provides spare parts for all models of cars.<br/>Moreover, We deliver our products to you wherever you are and whenever you want.
+                    </p>
+                </div>
+                <div class="contact-text">
+                    <p>
+                        You are per E-mail away to get our services <br/> Please do not hesitate to contact us.! <br /><span>creeds@gmail.com</span>
+                    </p>
+                </div>
```

Comment:
- [ ] On the about page, the title section should be centered. kindly make the text alignment center.